### PR TITLE
Updated fee summary props

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* /bcgov/entity#6546

*Description of changes:*
- deleted isSummaryMode prop
- added confirmLabel prop

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/FeeSummary/FeeSummary.stories.ts [hunk 1]
```diff
@@ -27,6 +27,6 @@ Default.args = {
   payApiUrl: 'https://pay-api-dev.pathfinder.gov.bc.ca/api/v1/',
   hasConflicts: false,
   isBusySaving: false,
-  isSummaryMode: false,
+  confirmLabel: 'Continue',
```
Comment (severinbeauvais) [inline src/components/FeeSummary/FeeSummary.stories.ts:30]: For display in StoryBook.

File: src/components/FeeSummary/FeeSummary.vue [hunk 2]
```diff
@@ -73,19 +73,14 @@ export default class FeeSummary extends Vue {
   @Prop({ default: false })
   readonly isBusySaving: boolean
 
-  /** Indicator that the client is in Summary mode. */
-  @Prop({ default: false })
-  readonly isSummaryMode: boolean
+  /** Label for Confirm button. */
+  @Prop({ default: 'Confirm' })
+  readonly confirmLabel: boolean
```
Comment (severinbeauvais) [inline src/components/FeeSummary/FeeSummary.vue:78]: I included a reasonable default.

File: src/components/FeeSummary/FeeSummary.vue [hunk 3]
```diff
@@ -73,19 +73,14 @@ export default class FeeSummary extends Vue {
...
 
   /** Message to display if there is an error. */
   @Prop({ default: '' })
   readonly errorMessage: string
 
-  /** Returns the correct btn label according to state. */
-  private get confirmLabel (): string {
-    return this.isSummaryMode ? 'File and Pay' : 'Review and Certify'
```
Comment (severinbeauvais) [inline src/components/FeeSummary/FeeSummary.vue:86]: I was not certain that these are the only possible strings, so I changed the label to a prop,

**Review by cameron-freshworks — APPROVED** [review]
Looks good!

**Pull request merged**